Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -4160,6 +4160,21 @@ LoginStatusAPIEnabled:
     WebCore:
       default: false
 
+LoginStatusAPIRequiresAuthnEnabled:
+  type: bool
+  status: testable
+  category: dom
+  humanReadableName: "Require Login Status API with WebAuthn"
+  humanReadableDescription: "Enable the proposed Login Status API which requires WebAuthn"
```

Comment:
This preference doesn't enable the Login Status API on its own, we should change the description to reflect that.